actions/test-app/action.yml

```diff
@@ -19,15 +19,14 @@ runs:
       run: |
         ## --------------------------------------------------------------------
         v <- paste0(version$major, ".", version$minor)
-        cat(paste0("::set-output name=version::", v, "\n"))
+        cat("version=", v, "\n", file = Sys.getenv("GITHUB_OUTPUT"), sep = "", append = TRUE)
 
     - name: Test app
       shell: Rscript {0}
-      working-directory: ${{ inputs.app-dir }}
       run: |
         ## --------------------------------------------------------------------
         options(crayon.enabled = TRUE)
-        shinytest2::test_app()
+        shinytest2::test_app("${{ inputs.app-dir }}")
 
     - name: Show testthat output
```

Comment:
Does this step actually work given that we haven't run `R CMD check` (and thus haven't produced a `check/` folder)?

Reply:
Ha. No it does not. 😭 Thank you!

The output is already displayed in the step before this. 
**Removing this step

------------------------------

<details>

<summary>Ex from <a href="https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:132">https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:132</a></summary>

```
Run rstudio/shinytest2/actions/test-app@v1
  with:
    app-dir: tests/testthat/apps/hello
    upload-snapshots: false
  env:
    GITHUB_PAT: ***
    R_KEEP_PKG_SOURCE: yes
    R_LIBS_USER: /home/runner/work/_temp/Library
    TZ: UTC
    _R_CHECK_SYSTEM_CLOCK_: FALSE
    NOT_CRAN: true
    RSPM: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    RENV_CONFIG_REPOS_OVERRIDE: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    R_LIB_FOR_PAK: /opt/R/4.[2](https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:2).2/lib/R/site-library
    MSYS: winsymlinks:nativestrict
Run ## --------------------------------------------------------------------
  ## --------------------------------------------------------------------
  v <- paste0(version$major, ".", version$minor)
  cat(paste0("::set-output name=version::", v, "\n"))
  shell: /usr/local/bin/Rscript {0}
  env:
    GITHUB_PAT: ***
    R_KEEP_PKG_SOURCE: yes
    R_LIBS_USER: /home/runner/work/_temp/Library
    TZ: UTC
    _R_CHECK_SYSTEM_CLOCK_: FALSE
    NOT_CRAN: true
    RSPM: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    RENV_CONFIG_REPOS_OVERRIDE: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    R_LIB_FOR_PAK: /opt/R/4.2.2/lib/R/site-library
    MSYS: winsymlinks:nativestrict
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Run ## --------------------------------------------------------------------
Loading required package: testthat
Loading required package: shiny
✔ | F W S  OK | Context

⠏ |         0 | app-click                                                       
⠋ |         1 | app-click                                                       
⠙ |         2 | app-click                                                       
⠴ |         6 | app-click                                                       
✔ |         8 | app-click [8.5s]

⠏ |         0 | app-init-args                                                   
⠋ |         1 | app-init-args                                                   
⠙ |         2 | app-init-args                                                   
✔ |         2 | app-init-args [1.8s]

⠏ |         0 | app-variant                                                     
⠋ |         1 | app-variant                                                     
✔ |         2 | app-variant [1.7s]

⠏ |         0 | app                                                             
⠋ |   1     0 | app                                                             
⠹ |   2     1 | app                                                             
⠏ |   6     4 | app                                                             
⠹ |   8     5 | app                                                             
✔ |   8     9 | app [[3](https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:3).7s]
────────────────────────────────────────────────────────────────────────────────
Warning (test-app.R:20): basic website example works using shinytest
Adding new file snapshot: 'tests/testthat/_snaps/linux-[4](https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:4).2/001.png'

Warning (test-app.R:21): basic website example works using shinytest
Adding new file snapshot: 'tests/testthat/_snaps/linux-4.2/002_.png'

Warning (test-app.R:21): basic website example works using shinytest
Adding new file snapshot: 'tests/testthat/_snaps/linux-4.2/002.json'

Warning (test-app.R:24): basic website example works using shinytest
Adding new snapshot for variant 'linux-4.2':
"Hello Hadley!"

Warning (test-app.R:2[5](https://github.com/rstudio/shinytest2/actions/runs/4107351632/jobs/7086722315#step:6:5)): basic website example works using shinytest
Adding new snapshot for variant 'linux-4.2':
"<div id=\"greeting\" class=\"shiny-text-output shiny-bound-output\" aria-live=\"polite\">Hello Hadley!<\/div>"

Warning (test-app.R:28): basic website example works using shinytest
Adding new file snapshot: 'tests/testthat/_snaps/linux-4.2/003_.png'

Warning (test-app.R:28): basic website example works using shinytest
Adding new file snapshot: 'tests/testthat/_snaps/linux-4.2/003.json'

Warning (test-app.R:41): basic website example works using testthat
Adding new file snapshot: 'tests/testthat/_snaps/linux-4.2/manual-screenshot.png'
────────────────────────────────────────────────────────────────────────────────

⠏ |         0 | execute-js                                                      
⠋ |         1 | execute-js                                                      
✔ |         6 | execute-js [1.6s]

══ Results ═════════════════════════════════════════════════════════════════════
Duration: 17.4 s

[ FAIL 0 | WARN 8 | SKIP 0 | PASS 27 ]
Run ## --------------------------------------------------------------------
  ## --------------------------------------------------------------------
  echo ::group::Show testthat output
  find check -name 'testthat.Rout*' -exec cat '{}' \; || true
  echo ::endgroup::
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GITHUB_PAT: ***
    R_KEEP_PKG_SOURCE: yes
    R_LIBS_USER: /home/runner/work/_temp/Library
    TZ: UTC
    _R_CHECK_SYSTEM_CLOCK_: FALSE
    NOT_CRAN: true
    RSPM: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    RENV_CONFIG_REPOS_OVERRIDE: https://packagemanager.posit.co/cran/__linux__/jammy/latest
    R_LIB_FOR_PAK: /opt/R/4.2.2/lib/R/site-library
    MSYS: winsymlinks:nativestrict
Show testthat output
  find: ‘check’: No such file or directory
```

</details>

Reply:
bfb0c2d